includes/class-wc-tracker.php

```diff
@@ -378,6 +395,24 @@ private static function get_orders() {
 		return array_merge( $order_dates, $order_counts, $order_totals );
 	}
 
+	/**
+	 * Get order counts by status.
+	 *
+	 * @param string $status The string of the order status without `wc-` prefix.
+	 * @return int $count The count of orders per status.
+	 */
+	public static function get_orders_count_by_status( $status = '' ) {
+		if ( empty( $status ) ) {
+			return 0;
+		}
+
+		$status = 'wc-' . $status;
+
+		$count = wp_count_posts( 'shop_order' );
```

Comment:
Nitpick, but I think it might be better to operate on a higher level of abstraction, i.e. use the infrastructure for counting orders if we decide to migrate to separate tables? Maybe `wc_orders_count` together with `wc_get_order_statuses`?